src/core/scene/metaTags.js

```diff
@@ -1,46 +1,78 @@
-module.exports = function initMetaTags (scene) {
-  if (!scene.isMobile) { return; }
-  injectMetaTags();
-};
+var extend = require('../../utils').extend;
+
+var MOBILE_HEAD_TAGS = module.exports.MOBILE_HEAD_TAGS = [
+  Meta({name: 'viewport', content: 'width=device-width,initial-scale=1,maximum-scale=1,shrink-to-fit=no,user-scalable=no'}),
```

Comment:
Just to make sure. Does this PR consider the viewport problem with the twitter appview?